.travis.yml

```diff
@@ -25,19 +25,23 @@ env:
   - VERSION="11.0" TESTS="0" LINT_CHECK="0"
   matrix:
   - LINT_CHECK="1"
-  - TESTS="1" ODOO_REPO="OCA/OCB"
-  - TESTS="1" ODOO_REPO="odoo/odoo"
+  - TESTS="1" ODOO_REPO="OCA/OCB" EXCLUDE="runbot_buildout"
+  - TESTS="1" ODOO_REPO="OCA/OCB" EXCLUDE="runbot_travis2docker"
 
 before_install:
   # Fix https://github.com/travis-ci/travis-ci/issues/8982#issuecomment-354357640
   - python3.5 -c "import fcntl; fcntl.fcntl(1, fcntl.F_SETFL, 0)"
 
 install:
+  - pip install zc.recipe.egg --no-cache-dir
```

Comment:
Did you try using requirements.txt file?

Reply:
i'll try now